pennylane/devices/default_qubit.py

```diff
@@ -149,7 +150,15 @@ def __init__(self, wires, *, shots=1000, analytic=True, cache=0):
     def map_wires(self, wires):
         # temporarily overwrite this method to bypass
         # wire map that produces Wires objects
```

Comment:
Perhaps better to add more details on what needs to be done long term here? E.g., `# TODO:...`